scripts/zones/North_Gustaberg_[S]/mobs/Olgoi-Khorkhoi.lua

```diff
@@ -6,8 +6,17 @@ require("scripts/globals/hunts")
 -----------------------------------
 local entity = {}
 
+entity.onMobSpawn = function(mob)
+    mob:setRespawnTime(0)
```

Comment:
Looking around the codebase, I don't think setting the respawn time is required - I think it's implied?

Possibly depending on what the flags in the db say for the mob